configuration/src/lib.rs

```diff
@@ -336,8 +336,11 @@ impl NomadConfig {
     }
 
     /// Get a reference to the nomad config's gas map.
-    pub fn gas(&self) -> &HashMap<String, NomadGasConfig> {
-        &self.gas
+    pub fn gas(&self) -> HashMap<String, &NomadGasConfig> {
```

Comment:
is there a reason not to be 

```
    pub fn gas(&self) -> &HashMap<String, NomadGasConfigs> {
```

Reply:
My thought was that people don't care about `NomadGasConfig[s]`, only the inner type

Reply:
hold on one sec, gonna try to get rid of the newtype wrapper